http/http_service.go

```diff
@@ -61,13 +61,21 @@ func (httpSvc *HttpService) RegisterSharedRoutes(e *echo.Echo) {
 	e.HideBanner = true
 
 	e.Use(middleware.RequestLoggerWithConfig(middleware.RequestLoggerConfig{
-		LogURI:    true,
-		LogStatus: true,
+		LogURI:       true,
+		LogStatus:    true,
+		LogRemoteIP:  true,
+		LogUserAgent: true,
+		LogHost:      true,
+		LogRequestID: true,
 		LogValuesFunc: func(c echo.Context, values middleware.RequestLoggerValues) error {
 			logger.Logger.WithFields(logrus.Fields{
-				"uri":    values.URI,
-				"status": values.Status,
-			}).Debug("handled echo request")
+				"uri":        values.URI,
+				"status":     values.Status,
+				"remote_ip":  values.RemoteIP,
```

Comment:
on cloudflare this would likely be the cloudflare IP and one would like the connecting_ip from the CF- header

Reply:
I will test this with the fly gateway